cypress/cypress/component/AutoSubmit/AutoSumbit.cy.tsx

```diff
@@ -29,25 +29,25 @@ it("radio button multiple autosave works", () => {
   );
 
   cy.get("@onSubmitSpy1").should("have.callCount", 0);
-  options1.forEach(({ value, label }, i) => {
+  for (const [i, { value, label }] of options1.entries()) {
```

Comment:
@neotob 

I need to get used to this syntax but i see the benefits so it's fine by me :-)

Reply:
same for me, not convinced that this is the more beautiful way.. but i see the point of trying to have it the same everywhere